ec2api/api/instance.py

```diff
@@ -533,7 +534,6 @@ def _format_attr_disable_api_termination(result):
 
 def _format_reservation(context, reservation, formatted_instances, os_groups):
     return {
-        'reservationId': reservation['id'],
         'ownerId': reservation['owner_id'],
```

Comment:
I was thinking about the instancesSet response in DescribeInstances and RunInstances. The instancesSet comes under reservationSet, and which itself is form of a list. Should we show only the elements of this final list, and hide the ownerId etc information available at the top level.